### ISSUE TITLE
CrossOver MacOs

### ISSUE THREAD

**Issue by ninogh** [Perfare/UnityStudio#1017]
It is possible to use AssetStudioGUI in MacOs with CrossOver (BigSur, Intel)?

**Comment by TonyStarkBy** [comment]
Yes, but you will need to download and install Microsoft .NET runtime manually.
